modules/apps/portal-search/portal-search-web/src/main/resources/com/liferay/portal/search/web/internal/search/results/web/portlet/display/template/dependencies/portlet_display_template_list.ftl

```diff
@@ -16,7 +16,7 @@
 							<span class="sticker">
 								<span class="sticker-overlay">
 									<img
-										alt="${languageUtil.get(locale, "thumbnail")}"
+										alt="${entry.getContent()!''}"
```

Comment:
No change, but just wanted to voice a concern:

This does follow how the image viewer sets `alt` https://github.com/liferay/liferay-portal/blob/master/modules/apps/document-library/document-library-preview-image/src/main/resources/META-INF/resources/preview/view.jsp#L46, but I'm still iffy on leaving this blank when there isn't a description provided. 

I'd prefer a fallback like "{title} thumbnail", but I think this is fine since the thumbnail is more of a visual helper anyway.

Reply:
I think you're right, maybe existing users might be a little surprised since originally it had at least a text like "thumbnail" at least... I made an update to have that `{title} thumbnail` text but lmk if that should be tweaked!